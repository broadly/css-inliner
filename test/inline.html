<!doctype>
<html>
  <head>
    <title>Hello world</title>
    <link rel="stylesheet" href="inline.style.css">
  </head>
  <body>
    <h1>Hello world</h1>
    <p>Testing out CSS inlining</p>
    <p style="font-size:9pt">Small print</p>
    <p style="font-family:Courier">Same font</p>
    <!--[if mso]> some code <![endif]-->
  </body>
</html>
