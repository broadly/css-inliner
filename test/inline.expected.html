<!doctype>
<html style="font-family:'Helvetica'">
  <head><style>p:hover{color:blue !important}@media print{h1{font-weight:bolder !important}}</style>
    <title>Hello world</title>
    
  </head>
  <body style="font-family:'Helvetica'">
    <h1 style="font-family:'Helvetica';font-size:64pt">Hello world</h1>
    <p style="font-family:'Helvetica';font-size:14pt">Testing out CSS inlining</p>
    <p style="font-size:9pt;font-family:'Helvetica'">Small print</p>
    <p style="font-family:'Helvetica';font-size:12pt">Same font</p>
    <!--[if mso]> some code <![endif]-->
  </body>
</html>
